compiler-plugin/build.gradle.kts

```diff
@@ -97,24 +84,18 @@ tasks.test {
 
     useJUnitPlatform()
 
-    systemProperty("idea.is.unit.test", "true")
```

Comment:
where did this system property get moved to? and line 102, 103? or was it removed/

Reply:
There's probably a reason these are there.